src/js/views/admin/shared/clinicians_components.js

```diff
@@ -63,6 +66,107 @@ const RoleComponent = Droplist.extend({
   },
 });
 
+const GroupsItemView = View.extend({
+  tagName: 'li',
+  className: 'clinician-groups__item',
+  template: hbs`<span class="clinician-groups__item-icon">{{far "users"}}</span>{{name}}<button class="button--icon clinician-groups__remove js-remove">{{far "trash-alt"}}</button>`,
+  triggers: {
+    'click .js-remove': 'remove',
+  },
+});
+
+const ClinicianGroupsView = CollectionView.extend({
+  tagName: 'ul',
+  childView: GroupsItemView,
+  childViewTriggers: {
+    'remove': 'remove',
+  },
+  onRemove(child) {
+    this.triggerMethod('remove:clinician', child.model);
+  },
+});
+
+const GroupsDropList = Droplist.extend({
+  initialize() {
+    this.listenTo(this.collection, 'update', this.show);
+  },
+  viewOptions: {
+    className: 'button-secondary clinician-groups__droplist',
+    template: hbs`{{far "users"}}Add Group...`,
+  },
+  picklistOptions: {
+    attr: 'name',
+  },
+  onShow() {
+    if (this.getOption('clinician').isNew()) {
```

Comment:
this seems ok, but after looking at it.. I think maybe the `listenTo` and the state setting should happen up in the parent component and just redo the droplist component.. otherwise modifying the state on the parent would have no effect.  ie: `myGroupsComponent.setState('disabled', true)`  _or_ we just don't pass the state down at all.  But right now we're only supporting the initial state which seem incomplete.

Reply:
I'd lean toward keeping the state self-contained within the Droplist component. Passing the state into it was extraneous anyway, it wasn't utilizing it at all.